src/collections/programs/layer5/index.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: 'Layer5'
+thumbnail: ./layer5.svg
+permalink: "https://layer5.io/careers/internships"
```

Comment:
Okay, this can be updated to `permalink: "../careers/internships/`,
and also add another field `redirect-from: "../careers/internships"`.

The final code of this file should be this:
```
---
title: "Layer5"
thumbnail: ./layer5.svg
permalink: "../careers/internships/"
redirect-from: "../careers/internships"
published: true
---
```
And, the work will be done.

Reply:
Okay, thank you very much. I do that now